sdk/eventhub/tests.data.yml

```diff
@@ -7,4 +7,5 @@ extends:
     ServiceDirectory: eventhub
     SDKType: data
     TimeoutInMinutes: 240
```

Comment:
Leaving this value as-is;  it is needed to ensure runs don't timeout in the China cloud; it could be lowered for others.